### PR TITLE
created trainer.py

### PR DESCRIPTION
**Pull request by brunolajoie**
- created `trainer.py`

### CODE REVIEW

File: lekuto/trainer.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Trainer():
+    def train(self):
+        pass
```
Comment (brunolajoie) [inline lekuto/trainer.py:3]: why passs here?

Comment (krokrob) [inline lekuto/trainer.py:3]: Nice start @brunolajoie , keep going!

Comment (brunolajoie) [reply lekuto/trainer.py:3]:
```suggestion
        return None
```

**Review by krokrob — APPROVED** [review]
Nice kick off, what's next?

**Pull request merged**